### ISSUE TITLE
TLS/SSL handshake failed: sslv3 alert handshake failure

### ISSUE THREAD

**Issue by AMD-NICK** [lunarmodules/copas#115]
I try to query https://www.zpool.ca/api/status or https://poll.gmod.app, but get a `copas.lua:688: TLS/SSL handshake failed: sslv3 alert handshake failure`

Copas version is 3.0.0

When I try to make the request directly through luasec 1.0.2 it will succeed

The sites is behind CloudFlare and on the Internet it is often referred to.

I found this issues, but I don't know about **SNI** and all these protocols: https://github.com/brunoos/luasec/issues/111, #109 

I also tried to set some params as in the luasec by default, but it doesn't work for me

```
reqt.protocol = "any"
reqt.options = {"all", "no_sslv2", "no_sslv3", "no_tlsv1"}
reqt.verify = "none"
```

Perhaps a related discussion, though not related to copas and luasec: https://forum.defold.com/t/ssl-handshake-failed-1-for-certain-https-hosts-def-2633-solved/7067/25

**Comment by AMD-NICK** [comment]
Fixed in this commit: 

https://github.com/fedenunez/copas/commit/45476aaabd5f00950d2c3f3f6dfb2541526b20f7 by @fedenunez

**Comment by Tieske** [comment]
fixed by #126
